content/en/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -35,7 +35,7 @@ For more information, see the
 
 Istio uses the Envoy proxy to generate metrics and provides its configuration in
 the `EnvoyFilter` at
-[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml).
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml).
```

Comment:
Should we use {{< istio_version >}} so this changes with release?

Reply:
> There are still many references to 1.8 in this doc. Do you want to not tackle those in this PR?
> 
> <img alt="image" width="316" src="https://user-images.githubusercontent.com/5502967/109088215-00e83500-76dd-11eb-8570-45fffe48e9d3.png">

Adding @richardwxn as he seems to have updated s/1.6/1.8/ in #8339 and may have opinions.

(All **I** wanted to do was remove the Experimental!)

Reply:
@craigbox we have to update all the references to 1.8 to 1.9 in this page